samples/tab-user-profile/ts/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-user-profile-ts",
+    "source": "officeDev",
+    "title": "Starter solution: Personal tab in TypeScript",
+    "shortDescription": "Create React App personal tab in TypeScript",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-user-profile/ts",
+    "longDescription": [
+      "Create React App personal tab in TypeScript"
+    ],
+    "creationDateTime": "2023-06-19",
+    "updateDateTime": "2023-06-19",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": ""
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": ""
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "tab"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/tab-user-profile/ts/Images/meta-OS-profileGif.gif",
+        "alt": "Solution UX showing personal tab for user profile in outlook"
```

Comment:
Solution UX shows how to open a user profile information card in Outlook using personal tab app.

Reply:
Fixed